services/resource-usage-tracker/requirements/_test.in

```diff
@@ -0,0 +1,25 @@
+#
+# Specifies dependencies required to run 'services/api-server/test'
+#  both for unit and integration tests!!
+#
+--constraint ../../../requirements/constraints.txt
+--constraint ./constraints.txt
+
+# Adds base AS CONSTRAINT specs, not requirement.
+#  - Resulting _text.txt is a frozen list of EXTRA packages for testing, besides _base.txt
+#
+--constraint _base.txt
+
+asgi-lifespan
+coverage
+faker
+httpx
+pytest
+pytest-asyncio
+pytest-cov
+pytest-mock
+pytest-runner
+pytest-sugar
+python-dotenv
+requests-mock
```

Comment:
What is the advantage of `requests-mock` vs `respx` (typical mock library in osparc-simcore for fastapi-based ) ?

Reply:
the prometheus-api-client uses requests